src/io/internal/pipelines/image/ReadDICOM/ReadImageDICOMFileSeries.cxx

```diff
@@ -257,11 +261,21 @@ int runPipeline(itk::wasm::Pipeline & pipeline, std::vector<std::string> & input
     }
 
     reader->SetFileNames(fileNames);
-    }
+  }
   else
-    {
+  {
     reader->SetFileNames(inputFileNames);
+  }
+
+  // copy sorted filenames as additional output
+  if(!outputFilenamesOption->empty())
+  {
+    auto finalFileList = reader->GetFileNames();
+    for (auto f = finalFileList.begin(); f != finalFileList.end(); ++f)
+    {
+      outputFilenames.Get() << *f << ":";
```

Comment:
Non-NTFS filenames can have a colon. Let's use a forward slash instead, since the majority of filesystems do not allow `/` in the filename.

Reply:
Changed to use null char '\0' in place of ':'.